modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+
+In the {logging} 5.8 and later versions, the Loki Operator supports managing access to logs through cluster wide policies, or by namespace. Users who are members of any group specified in the `adminGroups` field are considered admins in the authorization process. Admin access to logs is not restricted by namespace.
+
+.Example admin group settings
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  tenants:
+    mode: openshift-logging # <1>
+    openshift:
+      adminGroups: # <2>
+      - cluster-admin
+      - custom-admin-group
+----
+<1> Custom admin groups are only available in this mode.
+<2> Overrides the default groups (`system:cluster-admin`, `cluster-admin`, `dedicated-admin`)
```

Comment:
```suggestion
<2> Overrides the default groups (`system:cluster-admins`, `cluster-admin`, `dedicated-admin`)
```

---

Does it make sense to also mention that it's possible to disable the admin groups by providing an empty list:

```
adminGroups: []
```